jsonschema/scripts/codegen.rb

```diff
@@ -214,6 +214,33 @@ def format_description(raw_description, indent_string: "")
   end
 end
 
+class Dotnet < Codegen
+  def initialize(paths)
+    language_type_by_schema_type = {
+      'integer' => 'long',
+      'string' => 'string',
+      'boolean' => 'bool',
+    }
+
+    super(paths, language_type_by_schema_type)
+  end
+
+  def array_type_for(type_name)
+    "List<#{type_name}>"
+  end
+
+  def format_description(raw_description, indent_string: "")
```

Comment:
For now I've just got a couple of fixes to suggest here as the entire format is changing. But if you change this to single quotes and the item suggested below then that should be fine for now. Until the stuff gets merged

Reply:
indent_string is now ''